QUEST.tsv

```diff
@@ -2391,48 +2391,48 @@ QUEST_20150803_002398	There is a manager that manages the fortress of the great
+QUEST_20150918_002409	Welcome. Uska is waiting you in the central square of the Klaipeda. {nl}Well, is there anything that you are unsure of?
+QUEST_20150918_002410	But this is also my ordeal and mission that Laima has seen.. {nl}I will tell you Laima's message.{nl}
+QUEST_20150918_002411	Welcome! What do you need. {nl}Not like previous items, I made my best effort to collect.
+QUEST_20150918_002412	I heard that the Revelator appeared and saved the Mage Tower. {nl}I can't imagine what would have happened to Fedimian if the Mage Tower fell.. It's such a relief.{nl}Here, feel free to look around anytime.
+QUEST_20150918_002413	Fedimain used to be crowded pilgrims in the past. {nl}I heard someone protected the Cathedral so I'm hoping it will be lively as it was during it's glorious days.{nl}So, what kind of items are you looking for?
+QUEST_20150918_002414	The red haired wizard found the revelator to save the Mage Tower. {nl}Even when there were other Masters. He did save the Mage Tower so that was great but..{nl}Anyway, seeing is free so look around.
+QUEST_20150918_002415	Amazing, huh? The revelator who protected the Cathedral.{nl}So we decided to not increase the price even though we're in tough times. Take your time.
+QUEST_20150918_002416	Have you heard the rumor? Regarding the Mage Tower.{nl}A Revelator defeated the Demon Lord. {nl}If father was around, I could have given you some discount to celebrate.
+QUEST_20150918_002417	The people of Fedimian live by the teachings of the Maven.{nl}But the Demon Lord dirtied it ..The Revalators sure punished him for the Goddess. {nl}Now, what do you want me to check?{nl}
+QUEST_20150918_002418	I heard that some Revelator saved the Mage Tower. {nl}Fedimian should be safe now, right? {nl}Anyway, you might find some helpful items so look around. {nl}
+QUEST_20150918_002419	The expectation for the Revelator from all the people of Fedimian is rising. {nl}A news spread that a certain Revelator defeated the Demon Lord holding the Cathedral.{nl}Anyway, you don't ncesassrily have to buy something so feel free to look around.
+QUEST_20150918_002420	Seems like something is in that grass.
+QUEST_20150918_002421	What was she looking for even when she was dying? {nl}There's definitely a tragic story behind all this.
+QUEST_20150918_002422	Then something snapped. {n1}A fellow clergyman, I can't remember who, had murdered Dion in his sleep with a mace and I… {nl}Everyone… {nl}No one can remember why they wanted to become priest in the past anymore.
+QUEST_20150918_002423	Each Goddess Statue is known to have different abilities depending on which Goddess it depicts. {nl}For example, Goddess Zemyna gives blessing that enhaces the ability you want.
+QUEST_20150918_002424	But Ausurine Goddess staute of Klaipeda is a bit strange. {nl}The statue moves you to your desired area even though it doesn't depict Goddess Vakarine.
```

Comment:
But the statue of Goddess Ausrine in Klaipeda...

There is only one statue in Klaipeda